cli/config/configfile/file_test.go

```diff
@@ -176,7 +178,8 @@ func (c *mockNativeStore) Erase(registryHostname string) error {
 }
 
 func (c *mockNativeStore) Get(registryHostname string) (types.AuthConfig, error) {
-	return c.authConfigs[registryHostname], nil
+	println("hello it is me")
```

Comment:
![hello](https://user-images.githubusercontent.com/1804568/215726443-40019cfb-9d49-427e-aae2-ac6e6498c894.gif)

Reply:
hahahahaha. I noticed it and already removed it 😅